ang/civicase/ActivityIcon.html

```diff
@@ -0,0 +1,20 @@
+<span class="civicase__activity-icon" {{activity.icon}}>
+  <span>
+    <span
+      ng-if="!activity.icon && activity.category.indexOf('milestone') < 0"
+      ng-class="{'text-danger': activity.is_overdue}"
+      class="civicase__activity-no-icon">
+      {{activity.type[0]}}
+    </span>
+    <i
+      ng-if="activity.icon"
+      ng-class="{'civicase__activity-icon--ribbon': activity.category.indexOf('milestone') > -1, 'text-danger': activity.is_overdue}"
+      class="fa {{activity.icon}}"></i>
+
+    <i
+      ng-if="direction"
+      ng-class="{'text-danger': activity.is_overdue}"
+      class="civicase__activity-icon-arrow fa fa-arrow-{{ direction }}"></i>
```

Comment:
We should use only one set of icons app wide. Can we have a replacement of this in material icons?

Reply:
Same as above.